src/envs/cover.py

```diff
@@ -277,21 +280,23 @@ def _create_initial_state(self, blocks: List[Object],
                     break
             # [is_block, is_target, width, pose]
             data[target] = np.array([0.0, 1.0, width, pose])
-        # [hand]
-        data[self._robot] = np.array([0.0])
+        # [hand, pose_x, pose_z]
+        # For the non-PyBullet environments, pose_x and pose_z are constant.
+        data[self._robot] = np.array(
+            [0.5, self._workspace_x, self._workspace_z])
         state = State(data)
         # Allow some chance of holding a block in the initial state.
         if rng.uniform() < CFG.cover_initial_holding_prob:
             block = blocks[rng.choice(len(blocks))]
-            pick_pose = state.get(block, "pose")
+            block_pose = state.get(block, "pose")
+            pick_pose = block_pose
             if self._initial_pick_offsets:
                 offset = rng.choice(self._initial_pick_offsets)
                 assert -1.0 < offset < 1.0, \
                     "initial pick offset should be between -1 and 1"
                 pick_pose += state.get(block, "width") * offset / 2.
-            action = Action(np.array([pick_pose], dtype=np.float32))
```

Comment:
we were doing this in a stupid way before, this change shouldn't alter functionality. but the stupid way was breaking pybullet_cover because it doesn't have a self.simulate...